homeassistant/components/netatmo/climate.py

```diff
@@ -138,8 +146,8 @@ def __init__(self, data, room_id):
         except KeyError:
             _LOGGER.error("Thermostat in %s not available", room_id)
 
-        if self._module_type == NA_THERM:
-            self._operation_list.append(HVAC_MODE_OFF)
+        if self._module_type == NA_VALVE:
+            self._operation_list.remove(HVAC_MODE_OFF)
```

Comment:
You are changing a global const here.

Reply:
Oops, that is a reference. Sorry. Will fix that shortly.